server.js

```diff
@@ -2,23 +2,73 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
+import data from './data/netflix-titles.json'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+
+//setup connection to mongodb
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/shows"
```

Comment:
Good naming!

Reply:
@puj This is the right deployed link for the frontend part => https://react-movie-search-by-nasim.netlify.com/
Apparently I forgot to change the API-URL from localhost to Heroku :)